commit f1fa6b2baab4cc0bf44bd12812d007f9117f0cf0
Author: Lukáš Nykrýn <lnykryn@redhat.com>
Date:   Sat Oct 15 19:54:58 2016 +0200

    yummy:patchX:virt--add-possibility-to-skip-the-check-for-chroot---4374-.patch

virt: add possibility to skip the check for chroot (#4374)

    
    https://bugzilla.redhat.com/show_bug.cgi?id=1379852

diff --git a/src/basic/virt.c b/src/basic/virt.c
index dace1f4..007375c 100644
--- a/src/basic/virt.c
+++ b/src/basic/virt.c
@@ -33,6 +33,7 @@
 #include "string-table.h"
 #include "string-util.h"
 #include "virt.h"
+#include "env-util.h"
 
 static int detect_vm_cpuid(void) {
 
@@ -483,6 +484,9 @@ int detect_virtualization(void) {
 int running_in_chroot(void) {
         int ret;
 
+        if (getenv_bool("SYSTEMD_IGNORE_CHROOT") > 0)
+                return 0;
+
         ret = files_same("/proc/1/root", "/");
         if (ret < 0)
                 return ret;
